source/use-cases/sensitive-data-encryption.txt

```diff
@@ -124,7 +126,49 @@ In this step, we generate a local master key to expedite setup of our developmen
           }
 
 
-B. Define a JSON Schema
+.. _fle-create-a-data-encryption-key:
+
+B. Create a Data Encryption Key
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this section, we configure the MongoDB driver generate a data
```

Comment:
configure the MongoDB driver _**and/to**_ generate a data key...

Reply:
Oops, good catch, will fix.